util/namex-api.ts

```diff
@@ -183,3 +183,7 @@ export async function getCorporation(corpNum: string) {
 export async function getPayments(id: number) {
   return callNamexApi(getNamexApiUrl(`/payments/${id}`))
 }
+
+export async function getStats(params: string) {
+  return callNamexApi(getNamexApiUrl('/requests/stats' + params))
+}
```

Comment:
If the `URLSearchParams` suggestion in `stats.ts` was accepted, then this is what `getStats` would look like
```suggestion
export async function getStats(params: URLSearchParams) {
  return callNamexApi(getNamexApiUrl('/requests/stats?' + params))
}
```